package.json

```
@@ -58,7 +59,8 @@
     "eth-gas-reporter/colors": "1.4.0"
   },
   "dependencies": {
-    "@openzeppelin/contracts": "4.7.1"
+    "@openzeppelin/contracts": "4.7.1",
+    "merkletreejs": "^0.2.32"
```

```suggestion
    "merkletreejs": "0.2.32"
```
We usually set "exact" version on dependencies, just to be sure that everyone is working on the same version of packages (although `yarn.lock` should take care of that too).